maths/Kynea_numbers.dart

```diff
@@ -0,0 +1,27 @@
+// Function to calculate nth kynea number
```

Comment:
You could add some tests
```suggestion
import 'package:test/test.dart';

// Function to calculate nth kynea number
```